### ISSUE TITLE
Difference between the models in the repo

### ISSUE THREAD

**Issue by saunair** [eric-yyjau/pytorch-superpoint#78]
I see there are three versions, Superpoint (with no batch norm, assuming there's no training), SuperPointNet(With batchnorm) and SuperPointGauss(with group norm). 

The default in configs seem to be the one with group norm. Are there any performance differences between them?

**Comment by eric-yyjau** [comment]
Hi @saunair,

Thanks for your question.
I think SuperPointGauss2 uses batchnorm (pytorch-superpoint/SuperPointNet_gauss2.py at 4ff74df8fa3c10ce9eb9fdc561f787d9e8bc9691 · eric-yyjau/pytorch-superpoint
https://github.com/eric-yyjau/pytorch-superpoint/blob/4ff74df8fa3c10ce9eb9fdc561f787d9e8bc9691/models/SuperPointNet_gauss2.py).
The architecture should be the same as the original work.

Yes. I did experiment with group norm (https://github.com/eric-yyjau/pytorch-superpoint/blob/4ff74df8fa3c10ce9eb9fdc561f787d9e8bc9691/models/SuperPointNet.py). However, I saw no performance improvement on that. I observed similar performance.
